client/src/components/categorical/index.js

```diff
@@ -197,7 +198,8 @@ class Categories extends React.Component {
         )}
         {/* WRITEABLE FIELDS */}
         {allCategoryNames.map((catName) =>
-          schema.annotations.obsByName[catName].writable ? (
+          schema.annotations.obsByName[catName].writable &&
+          schema.annotations.obsByName[catName].categories.length > 1 ? (
```

Comment:
```suggestion
          schema.annotations.obsByName[catName].writable ? (
```

We still want to display single value user annos in the left sidebar